public/js/src/client.js

```diff
@@ -52,8 +64,8 @@ $(document).ready(function(){
 		}
 
 		// Appends the styles to the document.
-		head = document.head || document.getElementsByTagName('head')[0],
-		style = document.createElement('style');
+		var head = document.head || document.getElementsByTagName('head')[0],
```

Comment:
If these were *meant* to be global variables, prefix them with `window.`.  This indicates to future developers that you meant to do it, and also causes the code not to break in `use strict` mode.